modules/contact/src/main/kotlin/edu/stanford/spezi/modules/contact/ContactsList.kt

```diff
@@ -0,0 +1,16 @@
+package edu.stanford.spezi.modules.contact
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import edu.stanford.spezi.modules.contact.model.Contact
+
+@Composable
+fun ContactsList(contacts: List<Contact>) {
+    val scrollState = rememberScrollState()
+    Column(modifier = Modifier.verticalScroll(scrollState)) {
```

Comment:
Let's use `LazyColumn` which is scrollable by default and takes care of efficiently reusing the items - similar to `LazyVStack`. Also here, let's add the modifier parameter

Reply:
done